### PR TITLE
Teamspeak

### PR DESCRIPTION
**Pull request by Marcel-Baumgartner**
Added teamspeak and mindustry

### CODE REVIEW

**Comment by Marcel-Baumgartner** [comment]
If you accept this pr you have to change the docker image in the teamspeak image because its currently using my build.

**Comment by vilhelmprytz** [comment]
Thanks for the contribution! It looks like this PR includes the changes from #11. Should I close #11 and review both the industry image and TeamSpeak image in this PR?

**Comment by Marcel-Baumgartner** [comment]
Yes

File: images/other/mindustry.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  "installation": {
+    "docker_image": "wilfreddev/debian:stretch",
+    "shell": "/bin/bash",
+    "script": [
+      "apt update",
+      "apt install -y curl jq wget",
+      "GITHUB_PACKAGE=Anuken/Mindustry",
+      "MATCH=server-release.jar",
+      "cd /server",
+      "if [ -z ${DOWNLOAD_LINK} ]; then",
```
Comment (vilhelmprytz) [inline images/other/mindustry.json:25]: `$DOWNLOAD_LINK` seems to be undefined. Is it intended to be used for something?

Comment (vilhelmprytz) [reply images/other/mindustry.json:25]: I see. Alright, no rush :slightly_smiling_face:

**Comment by vilhelmprytz** [comment]
Have you tried using the TeamSpeak image? Wilfred binds the port using TCP, and TeamSpeak uses UDP, so I'm not able to login to the TeamSpeak server.

```bash
$ wilfred servers | grep teamspeak
amltj8md  teamspeak  teamspeak           1024    9987  None              running
$ docker container ls --all | grep amltj8md 
9939b0c7e6df   wilfreddev/debian:bullseye   "./ts3server default…"   3 minutes ago   Up 3 minutes   0.0.0.0:9987->9987/tcp, :::9987->9987/tcp   wilfred_amltj8md
```

**Comment by Marcel-Baumgartner** [comment]
I only tried to start it. Is there a way to add udp in the current version of wilfred?
Or can you make it so udp and tcp are bind both like in Pterodactyl?

**Comment by vilhelmprytz** [comment]
> I only tried to start it. Is there a way to add udp in the current version of wilfred? Or can you make it so udp and tcp are bind both like in Pterodactyl?

Not possible currently in Wilfred :slightly_frowning_face: https://github.com/wilfred-dev/wilfred/blob/47cf3f0cfdbbf95852fc7312f8c554c2374b9ae5/wilfred/api/servers.py#L394

Yes, binding both TCP and UDP would be doable. I created https://github.com/wilfred-dev/wilfred/issues/124.

**Comment by Marcel-Baumgartner** [comment]
Perfect

**Comment by Marcel-Baumgartner** [comment]
Soooo.... I removed finally the unused party of the mindustry image ;)

**Comment by vilhelmprytz** [comment]
Thanks! When the next version of Wilfred is released, it will bind to both TCP and UDP. I'll make sure to get this merged then.

**Comment by Marcel-Baumgartner** [comment]
Ok, thanks. Until then i'm working on an image for fivem 👍

**Comment by vilhelmprytz** [comment]
Teamspeak image tested and verified to work, thanks!

**Review by vilhelmprytz — APPROVED** [review]
Both images are now tested and ready to merge. Thanks!

**Pull request merged**